app/src/main/java/io/pslab/adapters/SensorLoggerListAdapter.java

```diff
@@ -133,6 +133,12 @@ public void onClick(DialogInterface dialog, int whichButton) {
                         }
                         LocalDataLog.with().clearSensorBlock(block.getBlock());
                         dialog.dismiss();
+                        if (LocalDataLog.with().getAllSensorBlocks().size() <= 0){
```

Comment:
Will the `size()` go below 0?

Reply:
I should not...I just used less than equal to make it fail safe